rust/ast-generator/rust.ungram

```diff
@@ -414,7 +414,7 @@ AsmClobberAbi = 'clobber_abi' '(' ('@string' (',' '@string')* ','?) ')'
 // option := "pure" / "nomem" / "readonly" / "preserves_flags" / "noreturn" / "nostack" / "att_syntax" / "raw"
 AsmOption = 'pure' | 'nomem' | 'readonly' | 'preserves_flags' | 'noreturn' | 'nostack' | 'att_syntax' | 'raw' | 'may_unwind'
 // options := "options(" option *("," option) [","] ")"
-AsmOptions = 'options' '(' AsmOption *(',' AsmOption) ','? ')'
+AsmOptions = 'options' '(' (AsmOption (',' AsmOption)* ','?) ')'
```

Comment:
is this a change on top of the upstream ungram? If that's the case, this might make it tricky to grab this file directly from there at some point in the future...

Reply:
I should push this upstream as a fix.  It looks like a bug to me.

Reply:
I'm still ok with merging this as it is :slightly_smiling_face:

Reply:
https://github.com/rust-lang/rust-analyzer/pull/18715

Reply:
Wow these rust-analyzer folk are quick, the upstream patch is already merged!